chrome_manifest_v3/views/popupView.js

```
@@ -1216,14 +1232,21 @@ Gmail2Trello.PopupView.prototype.showMessage = function (parent, text) {
         break;
       case 'clearCacheNow':
         $status.html('Clearing');
-        var hash = {};
+        let hash = {};
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)